src/Conference/AddConferenceForm.jsx

```diff
@@ -82,8 +84,9 @@ const Form = () => {
         required
         maxLength="20"
       />
-      <Input type="submit" value="submit" />
+      <Input type="submit" value="Submit" />
     </form>
   );
 };
-export default Form;
+const CookieAddConferenceForm = withCookies(AddConferenceForm);
```

Comment:
This can just be writen as `export default withCookies(AddConferenceForm);`

Reply:
This then allows you to import it in the other files with any name you want i.e. `AddConferenceForm` I wouldn't bother appending cookie at the front as the component doesn't care whether it needs cookies